src/main/scala/chess/format/pgn/Binary.scala

```diff
@@ -140,7 +163,15 @@ object Binary {
         case DropR(role, pos, check)                      => drop(role, pos, check)
       }) map (_.toByte)
 
-    def moves(strs: Iterable[String]): Array[Byte] = strs.flatMap(move).to(Array)
+    def plies(strs: Iterable[String]): Array[Byte] = strs.flatMap(ply).to(Array)
+
+    def actionStrs(strs: ActionStrs): Array[Byte] =
+      if (strs.size == 0 || strs.map(_.size).max == 1) plies(strs.flatten)
+      else plies(strs.toList.map(_.toList :+ "").flatten)
+
+    // TODO confirm we should be doing this
+    def removeFinalDelimiter(strs: List[String]): List[String] =
```

Comment:
This function seems to be used therefore we can remove it, and dot need to confirm if we should be doing it? Its also found in draughts and fairy